olca-app/src/org/openlca/app/editors/graphical/model/ExchangePart.java

```diff
@@ -11,6 +11,9 @@
 import org.openlca.app.editors.graphical.policy.CreateLinkPolicy;
 import org.openlca.app.editors.graphical.view.ExchangeFigure;
 
+/**
+ * Corresponding {@link org.eclipse.gef.EditPart} of {@link ExchangeNode}.
```

Comment:
I think such comments are not so helpful and can be outdated very quickly. The type-declaration already exactly says what is written in the comment. Better just add comments to things that really give some value.